packages/firestore/src/core/event_manager.ts

```diff
@@ -42,23 +42,34 @@ export interface Observer<T> {
   error: EventHandler<Error>;
 }
 
+/**
+ * Interface for view changes processed by the EventManager.
```

Comment:
I actually kind of liked a name that is not as specific to SyncEngine, since this interface only represents a subset of SyncEngine's functionality. To match the other platforms, I still went ahead and named it `SyncEngineListener`. I also moved its declaration to `sync_engine.ts`.